### PR TITLE
INT-7151 alleviate "permission denied" errors in logs

### PR DESCRIPTION
**Pull request by jflinchbaugh**
* set fs group, uid, gid of pod running the container

JIRA: https://issues.sonatype.org/browse/INT-7151
Build: https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/Helm3%20Charts/job/Feature%20Snapshot%20Builds/job/INT-7151-fs-perms/

### CODE REVIEW

File: charts/nexus-iq/README.md [hunk 1]
```diff
@@ -69,59 +69,60 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `persistence.logCSI`                           | A YAML block for defining CSI Storage Driver configuration for the Log PV. The entire block is taken as you write it. Should support _any_ csi driver that your cluster has installed.  | `nil`                                                       |
+| `persistence.affinity.nodeSelectorTerms`       | A YAML block for defining the affinity node selection. This block is taken as you write it.                                                                                             | `nil`                                                       |
+| `resources`                                    | Resource requests and limits for the IQ pod in the cluster.                                                                                                                             | See `values.yaml` for suggested minimum recommended values. |
+| `podSecurityContext`                           | `securityContext` for the whole pod.                                                                                                                                                    | Default matches the stock container.                        |
```
Comment (jflinchbaugh) [inline charts/nexus-iq/README.md:125]: here's the doc for the field we're fixing. the rest of the changes in this table is only whitespace.

File: charts/nexus-iq/values.yaml [hunk 2]
```diff
@@ -75,8 +75,13 @@ serviceAccount:
...
 
-podSecurityContext: {}
-#  fsGroup: 2000
+# Default the pods UID and GID to match the nexus3 container.
+# Customize or remove these values from the securityContext as appropriate for
+# your deployment environment.
+podSecurityContext:
+  runAsUser: 1000
+  runAsGroup: 1000
+  fsGroup: 1000
```
Comment (jflinchbaugh) [inline charts/nexus-iq/values.yaml:84]: here's the `fsGroup` that the customers want uncommented all the time.

**Review by eduard-tita — APPROVED** [review]
LGTM

**Comment by nblair** [comment]
@jflinchbaugh can you comment on how this relates to #226, perhaps on that issue/PR?

**Pull request merged**